src/tests/nativeaot/SmokeTests/Exceptions/GenericExceptions.cs

```diff
@@ -0,0 +1,116 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+
+public class GenericExceptions
```

Comment:
The point of SmokeTests is that there is very limited number of them and they run very fast. It is not worth it to have dedicated smoketest for GenericExceptions.

Could you please add one case for the generic exceptions to the existing src\tests\nativeaot\SmokeTests\Exceptions smoke test?

Reply:
Sure, makes sense